src/network/networkd-radv.c

```diff
@@ -85,5 +85,14 @@ int radv_configure(Link *link) {
                         return r;
         }
 
+        if (link->network->router_search_domains) {
+                r = sd_radv_set_dnssl(link->radv,
```

Comment:
Is it safe to call this one without freeing `link->radv->dnssl` first?

Reply:
link->radv->dnssl is private to radv. sd_radv_set_dnssl frees the previous dnssl when set.

Reply:
I'm sorry. I somehow missed `free(ra->dnssl)` in `sd_radv_set_dnssl`. I should have read the code more carefully.